servers/zts/src/main/java/com/yahoo/athenz/zts/cert/impl/DynamoDBCertRecordStoreConnection.java

```diff
@@ -267,13 +267,8 @@ public int deleteExpiredX509CertRecords(int expiryTimeMins) {
     public List<X509CertRecord> updateUnrefreshedCertificatesNotificationTimestamp(String lastNotifiedServer,
                                                                       long lastNotifiedTime,
                                                                       String provider) {
-        try {
-            List<Item> items = getUnrefreshedCertsRecords(lastNotifiedTime, provider);
```

Comment:
Removed the try / catch as the catch could never happen. The inner methods already catch the exceptions